web_development_101/project_js_jquery.md

```diff
@@ -356,4 +356,5 @@ If you get totally stuck, solutions from other students are listed below.  I'd e
 * [Nathan Minchow's solution](https://github.com/aspenner/sketchpad) | [View in Browser](https://htmlpreview.github.io/?https://github.com/aspenner/sketchpad/blob/master/sketchpad.html)
```

Comment:
Hi there. I found a few errors on your submission, please review the following error messages: 
* There should be no spaces between a closing bracket ']' and a opening parenthesis '('.